index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Quiz 1</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+        <article class="cards1">
+        <header>
+            <h1>German Chocolate Cake</h1>
+        </header>
+        <section>
+        <element>
```

Comment:
I think you were on the right track, but confused by the language on this requirement. When asking for three block level elements, we didn't mean a custom `<element>` tag, but something more like `<div>`,`<li>`, or `<p>` into which the images and text could be insterted.